src/components/tiger/tiger.tsx

```diff
@@ -0,0 +1,203 @@
+        }
+
+        if (verboseMode) {
+            args.push("-v");
+        }
+
+        CommandHelper.runCommandWithPkexec("tiger", args, handleProcessData, handleProcessTermination)
+            .then(({ output, pid }) => {
+                setLoading(false);
+                setOutput(output);
+                setAllowSave(true);
+                setPid(pid);
+            })
+            .catch((error) => {
+                setOutput(`Error: ${error.message}`);
+                setLoading(false);
+            });
+    };
+
+    const handleSaveComplete = () => {
+        setHasSaved(true);
+        setAllowSave(false);
+    };
+
+    const clearOutput = () => {
+        setOutput("");
+        setHasSaved(false);
+        setAllowSave(false);
+    };
+
+    return (
+        <RenderComponent
+            title={title}
+            description={description}
+            steps={steps}
+            tutorial={tutorial}
+            sourceLink={sourceLink}
+        >
+            {!loadingModal && (
+                <InstallationModal
+                    isOpen={opened}
+                    setOpened={setOpened}
+                    feature_description={description}
+                    dependencies={dependencies}
+                />
+            )}
+            <form onSubmit={form.onSubmit(onSubmit)}>
+                <Stack>
+                    {LoadingOverlayAndCancelButtonPkexec(loading, pid, handleProcessData, handleProcessTermination)}
+                    <TextInput
+                        label="Target IP/Hostname"
+                        required
+                        {...form.getInputProps("targetIP")}
+                        placeholder="e.g. 192.168.1.1"
+                    />
+                    <TextInput
+                        label="Audit Level"
+                        required
+                        {...form.getInputProps("auditLevel")}
+                        placeholder="e.g. 5"
+                    />
+                    <TextInput
+                        label="Report File"
+                        {...form.getInputProps("reportFile")}
+                        placeholder="e.g. /path/to/report.txt"
+                    />
+                    <TextInput
+                        label="Enable Modules"
+                        {...form.getInputProps("enableModules")}
+                        placeholder="e.g. passwd, fs"
+                    />
+                    <TextInput
+                        label="Exclude Modules"
+                        {...form.getInputProps("excludeModules")}
+                        placeholder="e.g. account, cron"
+                    />
+                    <Checkbox
+                        label="Verbose Mode"
+                        checked={verboseMode}
+                        onChange={(event) => setVerboseMode(event.currentTarget.checked)}
+                    />
+                    <Button type="submit">Start {title}</Button>
```

Comment:
Need to have save output component here. 
```suggestion
  {SaveOutputToTextFile_v2(output, allowSave, hasSaved, handleSaveComplete)}
                    <Button type="submit">Start {title}</Button>
```